<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>POS System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QTabWidget" name="mainTabWidget">
      <property name="font">
       <font>
        <family>Poppins</family>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="inventoryTab">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/grid.svg</normaloff>:/images/grid.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Inventory</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="searchLayout">
          <item>
           <widget class="QLabel" name="searchIcon">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/images/edit.svg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchLineEdit">
            <property name="placeholderText">
             <string>Search products...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="productsTableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addProductButton">
            <property name="text">
             <string>Add Product</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/plus-circle.svg</normaloff>:/images/plus-circle.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editProductButton">
            <property name="text">
             <string>Edit Product</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/edit.svg</normaloff>:/images/edit.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteProductButton">
            <property name="text">
             <string>Delete Product</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/trash-2.svg</normaloff>:/images/trash-2.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="posTab">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/shopping-cart.svg</normaloff>:/images/shopping-cart.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Point of Sale</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QListView" name="posProductListView"/>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTableView" name="cartTableView"/>
            </item>
            <item>
             <widget class="QLabel" name="totalAmountLabel">
              <property name="font">
               <font>
                <family>Poppins</family>
                <pointsize>18</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Total: $0.00</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="completeSaleButton">
                <property name="text">
                 <string>Complete Sale</string>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/images/check-circle.svg</normaloff>:/images/check-circle.svg</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="cancelSaleButton">
                <property name="text">
                 <string>Cancel Sale</string>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/images/trash-2.svg</normaloff>:/images/trash-2.svg</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="reportsTab">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/bar-chart-2.svg</normaloff>:/images/bar-chart-2.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Reports</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="salesTableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="userManagementTab">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/user.svg</normaloff>:/images/user.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>User Management</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableView" name="usersTableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addUserButton">
            <property name="text">
             <string>Add User</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/plus-circle.svg</normaloff>:/images/plus-circle.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editUserButton">
            <property name="text">
             <string>Edit User</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/edit.svg</normaloff>:/images/edit.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteUserButton">
            <property name="text">
             <string>Delete User</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/trash-2.svg</normaloff>:/images/trash-2.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
